src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    getCategories: (state) => {
+      state.categories = 'Under Construction';
+    },
+  },
+});
+
+export default categoriesSlice.reducer;
+export const { getCategories } = categoriesSlice.actions;
```

Comment:
- [ ] In your `/src/redux/store.js` file, based on the project requirement, you are expected to create the store object using the `configureStore` method from the Redux/toolkit.  Kindly implement the following: 

    * Imports `configureStore `from `@reduxjs/toolkit`.
    * Imports `books` from `src/redux/books`.
    * Imports `categories` from `src/redux/categories`.
    * Use `configureStore` to create a reducer.
      * `books` and `categories` should be added to `configureStore` method.

Reply:
thanks @bobb-Rob but that was not listed in the requirements sorry for the oversight.

Project requirements
Add Redux Toolkit (npm install react-redux @reduxjs/toolkit).
Create a directory that will contain all your Redux logic (/src/redux)
Configure a Redux store (/src/redux/store.js)
Define a slice of state for books that:
Includes an array of books (initial state: empty array)
Includes a reducer that adds a book
Includes a reducer that removes a book
Define a slice of state for categories that:
Includes an array of categories (initial state: empty array)
Includes a reducer that checks the status and always returns "Under construction" (the initial state should check to that string)
Structure your application files using a "feature folder" approach and use the [ducks pattern](https://github.com/erikras/ducks-modular-redux) for your Redux files. Your files/folder structure could look like this:
/src
    |---/components
    |---/redux
        |--- /books
                | booksSlice.js
        |--- /categories
                | categoriesSlice.js
        | store.js
    | index.js
NOTE: No changes in React components are required in this project.